src/services/MethodDependencyBuilder.ts

```diff
@@ -51,9 +54,33 @@ export default class MethodDependencyBuilder {
             }
         }
 
+        this.populateMethodInterdependencies(headMethodsMap);
+
         return headMethodsMap;
     }
 
+    /**
+     * Populate method dependencies within the given method map (which methods depend on other methods)
+     *
+     * @param methodMap a dictionary from method identifier to method
+     */
+    private populateMethodInterdependencies(methodMap): void {
+        for (let methodIdentifier in methodMap) {
+            for (let otherMethodIdentifier in methodMap) {
+                if (methodIdentifier !== otherMethodIdentifier) {
+                    // Check contents of method for other method
+                    for (let contentLine of methodMap[methodIdentifier].content.split("\n")) {
```

Comment:
`O(n^3)` 👌

Reply:
uh excuse, me, its ```O(mn^2)``` where ```m``` is the most number of lines a method has. So it could be MUCH better, or MUCH worse

Reply:
Sounds good! ship it